src/commands/transaction/addressalias.ts

```diff
@@ -91,6 +98,14 @@ export default class extends ProfileCommand {
             throw new ExpectedError('You need to introduce namespace id.');
         }
 
+        options.maxFee = OptionsResolver(options,
+            'maxFee',
+            () => undefined,
+            'maxFee: ');
+        if (isNaN(parseInt(options.maxFee.toString())) || options.maxFee < 0 ){
```

Comment:
Max_Fee could be 0 if the node multiplier is set to 0 (applies to all files changed)

Reply:
I don't understand what ``isNaN(parseInt(options.maxFee.toString()))`` does.  I believe the cli will raise an error automatically since **maxFee** is defined as ``number``.

Reply:
I'm fix it

Reply:
Thank you for reminding me.